Makefile

```diff
@@ -1,321 +1,259 @@
-
+SCORECARD_TEST_CI_IMAGE = quay.io/operator-framework/scorecard-test
+SCORECARD_TEST_IMAGE ?= $(SCORECARD_TEST_CI_IMAGE)
+SCORECARD_TEST_ARCHES:="amd64" "ppc64le" "arm64" "s390x"
 image-build-scorecard-test:
-	./hack/image/build-scorecard-test-image.sh $(SCORECARD_TEST_BASE_IMAGE):dev
-
+	./hack/image/build-scorecard-test-image.sh $(SCORECARD_TEST_CI_IMAGE):dev
 image-push-scorecard-test:
-	./hack/image/push-image-tags.sh $(SCORECARD_TEST_BASE_IMAGE):dev $(SCORECARD_TEST_IMAGE)-$(shell go env GOARCH)
-
+	./hack/image/push-image-tags.sh $(SCORECARD_TEST_CI_IMAGE):dev $(SCORECARD_TEST_IMAGE)-$(shell go env GOARCH)
 image-push-scorecard-test-multiarch:
 	./hack/image/push-manifest-list.sh $(SCORECARD_TEST_IMAGE) ${SCORECARD_TEST_ARCHES}
 
-# Scorecard test kuttl image build/push.
 .PHONY: image-build-scorecard-test-kuttl image-push-scorecard-test-kuttl image-push-scorecard-test-kuttl-multiarch
-
+SCORECARD_TEST_KUTTL_CI_IMAGE = quay.io/operator-framework/scorecard-test-kuttl
+SCORECARD_TEST_KUTTL_IMAGE ?= $(SCORECARD_TEST_KUTTL_CI_IMAGE)
+SCORECARD_TEST_KUTTL_ARCHES:="amd64" "ppc64le" "arm64"
 image-build-scorecard-test-kuttl:
-	./hack/image/build-scorecard-test-kuttl-image.sh $(SCORECARD_TEST_KUTTL_BASE_IMAGE):dev
-
+	./hack/image/build-scorecard-test-kuttl-image.sh $(SCORECARD_TEST_KUTTL_CI_IMAGE):dev
 image-push-scorecard-test-kuttl:
-	./hack/image/push-image-tags.sh $(SCORECARD_TEST_KUTTL_BASE_IMAGE):dev $(SCORECARD_TEST_KUTTL_IMAGE)-$(shell go env GOARCH)
-
+	./hack/image/push-image-tags.sh $(SCORECARD_TEST_KUTTL_CI_IMAGE):dev $(SCORECARD_TEST_KUTTL_IMAGE)-$(shell go env GOARCH)
 image-push-scorecard-test-kuttl-multiarch:
 	./hack/image/push-manifest-list.sh $(SCORECARD_TEST_KUTTL_IMAGE) ${SCORECARD_TEST_KUTTL_ARCHES}
 
-##############################
-# Tests                      #
-##############################
-
-##@ Tests
+.PHONY: image-build-custom-scorecard-tests
+CUSTOM_SCORECARD_TESTS_CI_IMAGE = quay.io/operator-framework/custom-scorecard-tests
+CUSTOM_SCORECARD_TESTS_IMAGE ?= $(CUSTOM_SCORECARD_TESTS_CI_IMAGE)
+image-build-custom-scorecard-tests:
+	./hack/image/build-custom-scorecard-tests-image.sh $(CUSTOM_SCORECARD_TESTS_CI_IMAGE):dev
 
-# Static tests.
-.PHONY: test test-sanity test-unit
+##@ Test
 
-test: test-unit ## Run the tests
+.PHONY: test-all
+test-all: test-static test-e2e ## Run all tests
 
-test-sanity: tidy build/operator-sdk lint
-	./hack/tests/sanity-check.sh
+.PHONY: test-static
+test-static: test-sanity test-unit test-links ## Run all non-cluster-based tests
 
-test-unit: ## Run the unit tests
-	$(Q)go test -coverprofile=coverage.out -covermode=count -count=1 -short $(TEST_PKGS)
+.PHONY: test-sanity
+test-sanity: generate fix ## Test repo formatting, linting, etc.
+	git diff --exit-code # fast-fail if generate or fix produced changes
+	./hack/check-license.sh
+	./hack/check-error-log-msg-format.sh
+	go run ./hack/generate/changelog/gen-changelog.go -validate-only
```

Comment:
it is missing run the samples for we check that all is updated. See that the check has not been called: https://travis-ci.org/github/operator-framework/operator-sdk/jobs/735457416#L630

Reply:
Samples generation happens in the `generate` target, which is a make dependency of this `test-sanity` target.

It's called here: https://travis-ci.org/github/operator-framework/operator-sdk/jobs/735457416#L522